content/docs/command-reference/import.md

```diff
@@ -91,6 +110,9 @@ from the source repo.
   > [Importing and updating fixed revisions](#example-importing-and-updating-fixed-revisions)
   > example below).
 
+- `-b`, `--backup` - turns [backup mode](#backup-mode) on for the import,
+  so it can be pushed to remote storage (in case the original source is lost).
```

Comment:
What about `import-url`, does it have `--backup`? Probably should?

Reply:
`import-url` always made files in "backup mode", and still does. I haven't provided any option to disable backup mode for it though. I could do, or alternatively its behaviour could be documented.

Reply:
Ah yeah I was confused. No need to mention backup mode in import-url, as it's explained in the DVC metafiles doc.